### ISSUE TITLE
[ geography request ] - IHO Seas

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#5155]
### First Step: Explain what geography needs created.

All IHO Seas which can't be recovered from existing Arctos data

**Comment by dustymc** [comment]
[iho_seas.csv](https://github.com/ArctosDB/arctos/files/9767775/iho_seas.csv)

**Comment by sharpphyl** [comment]
Glad to see these added.  Some seas (Red Sea/Baltic Sea/Flores Sea) and gulfs are in the Ocean/Continent column. Does that matter with the new structure?

**Comment by dustymc** [comment]
>Does that matter with the new structure?

I don't think so. I tried to add the next thing up, whatever it might be, in the 'next thing up' field, continent-ocean, when there was some logical 'parent.' I didn't do that when there isn't a 'parent' or when there are multiples. I _think_ that will make things more clear and not add confusion now that we have a defined and finite scope (https://github.com/ArctosDB/arctos/issues/5138). 

I could easily be convinced that what I've done is confusing and/or clutter and these things can stand by themselves, which is more consistent with the source data.

**Comment by sharpphyl** [comment]
So, is this basically just the addition of spatial data for existing geographies or are you adding some geographies as well?  

For example, Gulf of Aqaba is in the list in the Red Sea.  In the past our records just had the Gulf of Aqaba as a specific locality and not in Seas or other higher geography.  Having it either way is fine (I like it in the higher geography, of course) as long as we know how to search for localities using it and other in the IHO list.

**Comment by dustymc** [comment]
I added the stuff in the spreadsheet, all IHO Seas should now be in Arctos and have spatial data. (Some will be near duplicates of other stuff, but if there was any doubt I went ahead and made them so there will be a migration path - https://github.com/ArctosDB/arctos/issues/5144) So a bit of both, but the result should be a complete dataset.

>how to search 

See https://github.com/ArctosDB/arctos/issues/5138 - at some level it doesn't matter what you assert (as long as it's not just wrong), geography is being calculated from locality so stuff should be findable (if there's a georeference, of course). But, consistent data is always nice, I'm happy to help move things around if there's a better fit with the current data.

For picking, all of these will contain ``spatial data source iho_world_seas`` in remarks, and anything which doesn't contain either that or ``spatial data source GADM`` (or any new source that we might agree to) will go away in some way (being deleted or being marked as a 'fun flexible feature') soon-ish. IHO should be complete, GADM is just getting started - I'm sure there will be bumps as I clean things up (perhaps after you've chosen then in data entry but before you've loaded), but I'll do whatever I can to make that as smooth as possible.
